README.md

```diff
@@ -36,6 +36,27 @@ Go to http://localhost:3000
 $ rails spec
 ```
 
+
+## Using Docker
+
+> NOTE: You'll need to have docker and docker-compose installed
+
+Build the points docker image
+```
+$ docker-compose build
+```
+
+Setup database and run migrations
+```
+$ docker-compose run web bundle exec rake db:setup
+$ docker-compose run web bundle exec rake db:migrate
```

Comment:
Not sure if it's better to do `rails db:setup` instead of `bundle exec rake db:setup`. I know the `rails` command already takes care of loading bundler and I think it does more things. Not too sure though, maybe it's worth the try?

Also, I think `db:setup` already migrates the db (I think it runs `db:schema:load` instead to make it faster) so the `db:migrate` is probably not needed.